src/Data/Object/Simulator.elm

```diff
@@ -28,14 +29,24 @@ type Results
         }
 
 
-availableProcesses : Db -> Query -> List Process
-availableProcesses { object } query =
+
+-- FIX: read the components from a file
+-- For now take the components from the example and consider that they are unique by name
```

Comment:
Ok to implement loading available components from a file in a later PR but I think we can safely and easily add a UUID field for components in this one.